client/src/client-only-routes/show-settings.tsx

```diff
@@ -171,36 +175,92 @@ export function ShowSettings(props: ShowSettingsProps): JSX.Element {
   return (
     <>
       <Helmet title={`${t('buttons.settings')} | freeCodeCamp.org`} />
-      <Container>
-        <main>
+      <div className='settings-container' id='settings-container'>
+        <SettingsSidebarNav
+          examTokenFlag={examTokenFlag}
+          userToken={userToken}
+        />
+        <main className='settings-main'>
           <Spacer size='l' />
-          <h1
-            id='content-start'
-            className='text-center'
-            style={{ overflowWrap: 'break-word' }}
-            data-playwright-test-label='settings-heading'
-          >
-            {t('settings.for', { username: username })}
-          </h1>
-          <MiscSettings
-            keyboardShortcuts={keyboardShortcuts}
-            sound={sound}
-            editorLayout={editorLayout}
-            resetEditorLayout={resetEditorLayout}
-            toggleKeyboardShortcuts={toggleKeyboardShortcuts}
-            toggleSoundMode={toggleSoundMode}
-          />
+          <ScrollElement name='username'>
+            <h1
+              id='content-start'
+              className='text-center'
+              style={{ overflowWrap: 'break-word' }}
+              data-playwright-test-label='settings-heading'
+            >
+              {t('settings.for', { username: username })}
+            </h1>
+          </ScrollElement>
           <Spacer size='m' />
-          <Privacy />
+          <ScrollElement name='account'>
+            <Account
+              keyboardShortcuts={keyboardShortcuts}
+              sound={sound}
+              editorLayout={editorLayout}
+              resetEditorLayout={resetEditorLayout}
+              toggleKeyboardShortcuts={toggleKeyboardShortcuts}
+              toggleSoundMode={toggleSoundMode}
+            />
+          </ScrollElement>
           <Spacer size='m' />
-          <Email
-            email={email}
-            isEmailVerified={isEmailVerified}
-            sendQuincyEmail={sendQuincyEmail}
-            updateQuincyEmail={updateQuincyEmail}
-          />
+          <ScrollElement name='privacy'>
+            <Privacy />
+          </ScrollElement>
+          <Spacer size='m' />
+          <ScrollElement name='email'>
+            <Email
+              email={email}
+              isEmailVerified={isEmailVerified}
+              sendQuincyEmail={sendQuincyEmail}
+              updateQuincyEmail={updateQuincyEmail}
+            />
+          </ScrollElement>
           <Spacer size='m' />
-          <Honesty isHonest={isHonest} updateIsHonest={updateIsHonest} />
+          <ScrollElement name='honesty'>
+            <Honesty isHonest={isHonest} updateIsHonest={updateIsHonest} />
+          </ScrollElement>
+          <Spacer size='m' />
+          {examTokenFlag && (
+            <>
+              <ScrollElement name='exam-token'>
+                <ExamToken />
+              </ScrollElement>
+              <Spacer size='m' />
+            </>
+          )}
```

Comment:
This probably came from a merge conflict.

In main, we don't put this component behind any rendering condition:

https://github.com/freeCodeCamp/freeCodeCamp/blob/5a703ef1e2dae65e5df4ec1061b80ae50ae6572a/client/src/client-only-routes/show-settings.tsx#L205

There is already an `ExamToken` component at line 265, so I think we can just wrap that one in ScrollElement and remove this instance.